doc/libpmemobj/pmemobj_open.3.md

```diff
@@ -166,10 +166,10 @@ perform the consistency check due to other errors.
 Not all file systems support **posix_fallocate**(3). _UW(pmemobj_create) will
 fail if the underlying file system does not support **posix_fallocate**(3).
 
-_WINUX(==q== On Windows if _UW(pmemobj_create) is called on an existing file
+_WINUX(=q= On Windows if _UW(pmemobj_create) is called on an existing file
 with FILE_ATTRIBUTE_SPARSE_FILE and FILE_ATTRIBUTE_COMPRESSED set,
 they will be removed, to physically allocate space for the pool.
-This is a workaround for _chsize() performance issues ==e==)
+This is a workaround for _chsize() performance issues =e=)
```

Comment:
`.` at the end of sentence and new line before SEE ALSO

Reply:
Done.